src/interface.js

```diff
@@ -0,0 +1,110 @@
+import Bowling from './bowling.js';
+
+var bowling;
+
+$("#start-game").click(() => {
+  startGame();
+});
+
+function startGame() {
+  initiateScorecard();
+  initiateGame();
+}
+
+function initiateScorecard() {
+  $(".game-over").remove();
+  $(".bowling-img").slideUp(); 
+  $(".roll").slideDown();
+  $(".scorecard").removeClass("hide");
+  $(".scorecard-body").empty();
+}
+
+function initiateGame() {
+  bowling = new Bowling();
+  refresh();
+}
+
+function refresh() {
+  setRollArea(bowling.currentFrame.pinsLeft);
+  $(".scorecard-body").empty();
+  let accumScore = 0;
+  for (let i = 0; i < bowling.frames.length; i++) {
+    let frame = bowling.frames[i];
+    accumScore += frame.score;
+    $(".scorecard-body").append(`
+    <div class="scorecard-item" id="frame-${i+1}">
+      <div class="frame-no" id="frame-no-${i+1}">${i+1}</div>
+      <div class="roll-1" id="roll-${i+1}-1">${frame.rolls[0] === undefined ? "" : frame.rolls[0]}</div>
+      <div class="roll-2" id="roll-${i+1}-2">${frame.rolls[1] === undefined ? "" : frame.rolls[1]}</div>
+      <div class="roll-3" ${i===9 ? "class='disable'" : ""} id="roll-${i+1}-3">${frame.rolls[2] === undefined ? "" : frame.rolls[2]}</div>
+      <div class="frame-score" id="${i+1}-score">${frame.score === 0 ? "" : accumScore}</div>
+      <div class="frame-result" id="${i+1}-result">${frame.result}</div>
+    </div>
+  `);
+  }
+}
+
+function setRollArea(pinsLeft) {
```

Comment:
This is really cool! You were adding or subtracting the number of images depending on the previous roll and then setting to 10 after a closed frame in the function nextRoll? Really cool!